server/lib/item-transform.js

```diff
@@ -0,0 +1,37 @@
+const get = require('lodash.get');
+
+const PRICE_MATCH = '[PRICE]';
+
+module.exports.itemTransform = item => {
```

Comment:
I'm wondering, would it makes sense to rename `item` to `offer`?
"Item" seems highly generic and someone without context would have no idea about what we are dealing with.

Reply:
I'd change that to `offerItemTransform`, WDYT?

Reply:
yes please! if it's not too much hassle. I think it will help make the code more readable.